extensions/ui-extension/src/targets/purchase/thank-you/block/render.tsx

```diff
@@ -1,14 +1,53 @@
 import { Community, Newsletter, GuideLinks } from '@/components/post-purchase';
 import { reactExtensionWithSentry } from '@/helpers/extension';
-import { BlockStack } from '@shopify/ui-extensions-react/checkout';
+import { isHeadlessStore } from '@/helpers/shop';
+import { useIfixitSettings } from '@/hooks/useIfixitSettings';
+import {
+   BlockStack,
+   useAttributeValues,
+   useAttributes,
+   useShop,
+} from '@shopify/ui-extensions-react/checkout';
+import { Shop } from '@shopify/ui-extensions/checkout';
+import { useEffect } from 'react';
+import * as Sentry from '@sentry/browser';
+import { useIfixitBaseUrl } from '@/hooks/useIfixitBaseUrl';
 
 export default reactExtensionWithSentry(
    'purchase.thank-you.block.render',
-   () => (
+   () => <ThankYou />
+);
+
+function ThankYou() {
+   useEffect(() => {
+      void useClearIfixitCart();
+   }, []);
+   return (
       <BlockStack>
          <Community />
          <Newsletter />
          <GuideLinks />
       </BlockStack>
-   )
-);
+   );
+}
+
+export const useClearIfixitCart = async () => {
+   const publicCartid = useAttributeValues(['publicCartid']);
```

Comment:
I think we need to destructure here
```suggestion
   const [publicCartid] = useAttributeValues(['publicCartid']);
```